tests/run-make/rust-lld-by-default-beta-stable/rmake.rs

```diff
@@ -12,7 +12,7 @@ use run_make_support::rustc;
 fn main() {
     // A regular compilation should not use rust-lld by default. We'll check that by asking the
     // linker to display its version number with a link-arg.
-    let output = rustc().link_arg("-Wl,-v").input("main.rs").run();
+    let output = rustc().arg("-Wlinker-messages").link_arg("-Wl,-v").input("main.rs").run();
```

Comment:
cc @rust-lang/release it continues to make me very uncomfortable to be editing tests that i have no way of running. if this fails in 6 weeks i do not know if i will have time to fix it.

Reply:
Locally, you can config `[rust] channel = "beta"` (or stable). For CI, you could temporarily change `src/ci/channel` for a `@bors try` attempt.

Reply:
This test runs in (full) CI on the x86_64-gnu-stable job:

https://github.com/rust-lang-ci/rust/actions/runs/12975734650/job/36187153380

2025-01-26T16:46:32.1849227Z test [run-make] tests/run-make/rust-lld-by-default-beta-stable ... ok